documentation.css

```diff
@@ -132,6 +132,14 @@ footer {
   width: 30rem;
 }
 
+#modal-component .modal {
+  z-index: 0;
```

Comment:
if z-index is 0 then no need to specifically provide one.

Reply:
Thanks for checking this, Hamza. Actually, I am overriding the z-index property here, default it has a z-index set and I am removing that.

Reply:
Okay good work himanshu bhai